project/pom.xml

```diff
@@ -807,7 +807,8 @@
               <compilerArg>-Xep:NonCanonicalStaticImport:ERROR</compilerArg>
               <compilerArg>-Xep:NonFinalCompileTimeConstant:ERROR</compilerArg>
               <compilerArg>-Xep:Overrides:ERROR</compilerArg>
-              <compilerArg>-Xep:PackageLocation:ERROR</compilerArg>
+              <!-- Disabled because of https://github.com/google/error-prone/issues/432 -->
+              <!-- <compilerArg>-Xep:PackageLocation:WARNING</compilerArg> -->
```

Comment:
I tried, but it says it cannot be disabled. Setting it to OFF or WARNING does not work.